.gitignore

```diff
@@ -2,4 +2,4 @@
 coverage.txt
 profile.out
 /monitor
-/rack
+/rack
```

Comment:
Can you please remove this change, might need to configure your editor to allow newlines at the end of a file.